src/main/kotlin/com/ampnet/projectservice/controller/pojo/response/ProjectResponse.kt

```diff
@@ -91,6 +91,11 @@ data class ProjectFullResponse(
     )
 }
 
+data class ProjectWithWalletFullResponse(
+    val project: ProjectFullResponse,
+    val wallet: WalletResponse?
```

Comment:
This is not correct response. See [task](https://dev.azure.com/ampnetio/AMPnet%20Crowdfunding/_boards/board/t/AMPnet%20Crowdfunding%20Team/Issues/?workitem=501). 
@mightymatth wanted response with one top object. 
```json
{
  "uuid": project-uuid,
  "name": project-name,
  ...
  "wallet": wallet-response-object
}
```